Ports/qt6-qtbase/package.sh

```diff
@@ -1,31 +1,31 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=qt6-qtbase
-version=6.2.3
-workdir=qtbase-everywhere-src-${version}
-useconfigure=true
-files="https://download.qt.io/official_releases/qt/$(cut -d. -f1,2 <<< ${version})/${version}/submodules/qtbase-everywhere-src-${version}.tar.xz qt6-qtbase-${version}.tar.xz 34d6d0072e197241463c417ad72610c3d44e2efd6062868e9a95283103d75df4"
-auth_type="sha256"
+port='qt6-qtbase'
+version='6.4.0'
+workdir='qtbase-everywhere-src-${version}'
+useconfigure='true'
+files="https://download.qt.io/official_releases/qt/$(cut -d. -f1,2 <<< ${version})/${version}/submodules/qtbase-everywhere-src-${version}.tar.xz qt6-qtbase-${version}.tar.xz cb6475a0bd8567c49f7ffbb072a05516ee6671171bed55db75b22b94ead9b37d"
+auth_type='sha256'
 configopts=(
     "-GNinja"
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
     "-DCMAKE_CROSSCOMPILING=ON"
     "-DQT_BUILD_TOOLS_WHEN_CROSSCOMPILING=ON"
-    "-DQT_HOST_PATH=/usr"
     "-DQT_FEATURE_cxx20=ON"
     "-DQT_FEATURE_ssl=ON"
     "-DQT_FEATURE_zstd=ON"
     "-DINPUT_opengl=no"
 )
-depends=("md4c" "zstd" "openssl")
-
-QT_HOST_PATH=/usr
-QT_HOST_CMAKE_PATH=${QT_HOST_PATH}/lib64/cmake
-QT_HOST_TOOLS="HostInfo CoreTools GuiTools WidgetsTools"
-QT_HOST_TOOLS_PATH="${QT_HOST_CMAKE_PATH}/Qt6%s/\n"
+depends=("double-conversion" "md4c" "openssl" "zstd")
 
 QT_DISABLED_FEATURES="sql opengl dbus systemsemaphore sharedmemory dnslookup"
 
 configure() {
+
+    QT_HOST_PATH="$(qmake6 -query QT_HOST_PREFIX)"
+    QT_HOST_CMAKE_PATH="$(qmake6 -query QT_HOST_LIBS)/cmake"
+    QT_HOST_TOOLS="HostInfo CoreTools GuiTools WidgetsTools"
+    QT_HOST_TOOLS_PATH="${QT_HOST_CMAKE_PATH}/Qt6%s/\n"
```

Comment:
I had to move these into configure() b/c the CI runners don't have qmake6 on them, for lint_ports.py :yaklkie: